src/main/java/com/efficio/fieldbook/web/common/controller/LocationsController.java

```diff
@@ -73,7 +51,7 @@ public Long getCurrentProgramID(final HttpServletRequest request) {
 		final String programUUID = this.contextUtil.getCurrentProgramUUID();
```

Comment:
As discussed, please check if you can use the location selector for Add Field module so we can delete this class already

Reply:
Done